docs/control-structures.rst

```diff
@@ -172,6 +194,9 @@ throws an exception or goes out of gas.
     external functions happen after any changes to state variables in your contract
     so your contract is not vulnerable to a reentrancy exploit.
 
+.. note::
```

Comment:
Can be merged in a separate PR.

Reply:
Ok, this came straight from the FAQ, but sure.